crates/top/re_sdk/src/recording_stream.rs

```diff
@@ -1160,20 +1163,32 @@ impl RecordingStream {
         self.log_serialized_batches_impl(row_id, ent_path, static_, comp_batches)
     }
 
-    /// Sets the recording properties.
-    ///
-    /// This is a convenience wrapper for statically logging to the entity path
-    /// that is reserved for recording properties.
+    /// Sends a property to the recording.
     #[inline]
-    pub fn set_properties(&self, properties: &RecordingProperties) -> RecordingStreamResult<()> {
-        self.log_static(EntityPath::partition_properties(), properties)
+    pub fn send_property<AS: ?Sized + AsComponents>(
+        &self,
+        name: impl Into<String>,
+        values: &AS,
+    ) -> RecordingStreamResult<()> {
+        let sub_path = EntityPath::from(name.into());
+        self.log_static(EntityPath::partition_properties().join(&sub_path), values)
     }
 
-    /// Sets the name of the recording.
+    /// Sends the name of the recording.
     #[inline]
-    pub fn set_name(&self, name: impl Into<String>) -> RecordingStreamResult<()> {
+    pub fn send_recording_name(&self, name: impl Into<String>) -> RecordingStreamResult<()> {
         let update = RecordingProperties::update_fields().with_name(name.into());
-        self.set_properties(&update)
+        self.log_static(EntityPath::recording_properties(), &update)
+    }
+
+    /// Sends the start time of the recording.
```

Comment:
since integers implicitely cast into `Timestamp` this should specify what time it is.
We should consider refining this in a follow-up: `set_time` makes the unit fairly explicit by the kinds of conversions it allows

```
rec.set_time("frame_nr", rerun::TimeCell::from_sequence(42));
rec.set_time("duration", std::time::Duration::from_millis(123));
rec.set_time("capture_time", std::time::SystemTime::now());
```
We should be able to have the `Timestamp` type to work similar to that 🤔